--- src/openvpn/Makefile.in.orig	2013-11-03 21:12:04.000000000 -0800
+++ src/openvpn/Makefile.in	2013-11-03 21:19:00.000000000 -0800
@@ -547,7 +547,7 @@
 
 openvpn$(EXEEXT): $(openvpn_OBJECTS) $(openvpn_DEPENDENCIES) $(EXTRA_openvpn_DEPENDENCIES) 
 	@rm -f openvpn$(EXEEXT)
-	$(AM_V_CCLD)$(LINK) $(openvpn_OBJECTS) $(openvpn_LDADD) $(LIBS)
+	libtool -static -o libopenvpn.a $(openvpn_OBJECTS)
 
 mostlyclean-compile:
 	-rm -f *.$(OBJEXT)
